dist/api_data.py

```diff
@@ -1067,8 +1067,8 @@ def __ge__(self, other):
         Config('chunk_size', '1MB', r'''
             size of cached chunks''',
             min='512KB', max='100GB'),
-        Config('storage_path', '', r'''
-            the absolute path to the file used as cache location'''),
+        Config('storage_path', 'WiredTigerChunkCache', r'''
+            the path (absolute or relative) to the file used as cache location'''),
```

Comment:
Why does this now have a default config? I understand the need to allow relative paths, but I'm not so sure about having this just end up on the same device as WiredTiger's home directory. If someone forgets to configure this, it could easily become a problem since we do a lot of IO on the volume with the chunk cache, which would kill performance for the rest of the system.

Reply:
Understood @korteland. My reasoning was that if the path is not provided we could default to the WT home directory. However, I am happy to get rid of this behavior.

That said, for our testing (python, workgen and format), as discussed earlier, we can make the file reside in the DB directory itself. Sound good?

Reply:
Yes, that seems like the best option for testing.

Reply:
Updated in [ca29077](https://github.com/wiredtiger/wiredtiger/pull/9589/commits/ca290772d2534e63ef0e0b6eab570252357672eb)